src/editor-sidebar/metadata-editor-modal.js

```diff
@@ -291,6 +293,33 @@ ${ __( 'Source', 'create-block-theme' ) }` }
 					}
 				/>
 
+				<TextareaControl
+					label={ __( 'Image Credits', 'create-block-theme' ) }
+					help={
+						<>
+							{ __(
+								'Credits for the images bundled with the theme.',
+								'create-block-theme'
+							) }
+							<br />
+							<ExternalLink href="hhttps://make.wordpress.org/themes/handbook/review/required/#1-licensing-copyright">
```

Comment:
```suggestion
							<ExternalLink
								href={ __(
									'https://make.wordpress.org/themes/handbook/review/required/#1-licensing-copyright',
									'create-block-theme'
								) }
							>
```

- Fix a typo (`hhttps`)
- Apply a translation function because the handbook page may be localized and the URL may change depending on the locale. Other external links will be fixed in #663.